integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/strategy/DpvStatusStrategy.java

```diff
@@ -82,7 +82,7 @@ public void checkStatus(Conversation conversation) {
             if (readStatus.isPresent()) {
                 ZonedDateTime readZoned = readStatus.get().getStatusDate().toGregorianCalendar().toZonedDateTime();
                 MessageStatus status = MessageStatus.of(lestStatus, readZoned.toLocalDateTime());
-                conversationService.registerStatus(conversation, status);
+                conversation  = conversationService.registerStatus(conversation, status);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "conversation". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1226)